apps/go-mod-validator/action.yaml

```diff
@@ -0,0 +1,81 @@
+name: go-mod-validator
+description: "Validates dependencies in go.mod files are tested"
+
+inputs:
+  # general inputs
+  checkout-repo:
+    description: "enable git checkout repo"
+    required: false
+    default: "true"
+  checkout-repo-fetch-depth:
+    description: "number of commits to fetch"
+    required: false
+    default: "0"
+  # inputs
+  github-token:
+    description: ""
+    required: true
+    default: ${{ github.token }}
+  # grafana inputs (optional)
+  metrics-job-name:
+    description: "grafana metrics job name"
+    required: false
+  metrics-id:
+    description:
+      "grafana metrics id, used for continuity of metrics during job name
+      changes  - required if metrics-job-name is passed"
+    required: false
+  gc-host:
+    description: "grafana hostname - required if metrics-job-name is passed"
+    required: false
+  gc-basic-auth:
+    description: "grafana basic auth - required if metrics-job-name is passed"
+    required: false
+  gc-org-id:
+    description:
+      "grafana org/tenant id - required if metrics-job-name is passed"
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - name: Checkout repo
+      if: inputs.checkout-repo == 'true'
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      with:
+        fetch-depth: ${{ inputs.checkout-repo-fetch-depth }}
+
+    - name: Checkout .github repo
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      with:
+        repository: "smartcontractkit/.github"
+        path: "github"
+
+    - name: Setup go
+      uses: actions/setup-go@cdcb36043654635271a94b9a6d1392de5bb323a7 # v5.0.1
+      with:
+        go-version: 1.22.4
+
+    - name: Setup Node
+      uses: actions/setup-node@1e60f620b9541d16bece96c5465dc8ee9832be0b # v4.0.3
+      with:
+        node-version: 20
+        node-version-file: package.json
+
+    - name: Run go-mod-validator
+      shell: bash
+      run:
+        GITHUB_TOKEN=${{ inputs.github-token }} node
+        github/apps/go-mod-validator/dist/index.js
```

Comment:
How is this working if `dist` hasn't been committed?

Reply:
Fixed it, I was testing with one of the previous commits